.github/workflows/release.yml

```diff
@@ -16,42 +16,14 @@ jobs:
   main:
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout repo
-        uses: actions/checkout@v3
-      - name: Setup node
-        uses: actions/setup-node@v3
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
         with:
           cache: 'yarn'
-      - name: Install deps
-        # this runs a build script so there is no dedicated build
-        run: yarn install
-      - name: Run storybook build script
-        run: yarn build-storybook
-  release:
-    needs: main
-    runs-on: ubuntu-latest
-    if: ${{ github.repository == 'pmndrs/drei' && contains('refs/heads/master,refs/heads/beta,refs/heads/alpha',github.ref) && github.event_name == 'push' }}
```

Comment:
there is actually no need to check:
- `contains('refs/heads/master,refs/heads/beta,refs/heads/alpha',github.ref)` because semantic-release won't release any branch that is not declared in `release.config.js`.`branches`
- `github.event_name == 'push'` because semantic-release will ignore PR: `"This run was triggered by a pull request and therefore a new version won't be published."`